benches/banking_stage.rs

```diff
@@ -39,6 +41,17 @@ fn check_txs(receiver: &Receiver<Vec<Entry>>, ref_tx_count: usize) {
     assert_eq!(total, ref_tx_count);
 }
 
+fn test_recorder(bank: &Arc<Bank>) -> (PohRecorder, Sender<WorkingBank>) {
```

Comment:
Can you prefix functions like this with `create_` as not to confuse them with tests?